infra/gcp/terraform/modules/workload-identity-service-account/outputs.tf

```
@@ -0,0 +1,25 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+output "service_account" {
+  description = "The serviceaccount that was created"
+  value       = google_service_account.serviceaccount
```

```suggestion
  value       = google_service_account.serviceaccount.email
```

Or do we want to get a boolean value ? (the description suggest something like that ?

---

I thought we might want access to the full service account resource vs. just its email

So far I've only been using email though, so I can refactor to just that